File: ReportManage/ReportManage/Data/Account.swift
//
//  Account.swift
//  ReportManage
//
//  Created by 葉家均 on 2022/12/11.
//

import Foundation

struct Account: Identifiable, Decodable {
    var id: Int
    var name: String
    var email: String
    var password: String
}

File: report_app/lib/data/account.dart
class Account {
  int id = 0;
  String name = "";
  String email = "";
  String password = "";

  Map<String, dynamic> toJson() => {
    'id': id,
    'name': name,
    'email': email,
    'password': password
  };

  Account.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json["name"],
        email = json["email"],
        password = json["password"];
}